cluster/gce/util.sh

```diff
@@ -616,7 +688,7 @@ function kube-down {
   local -a minions
   minions=( $(gcloud compute instances list \
                 --project "${PROJECT}" --zone "${ZONE}" \
-                --regexp "${INSTANCE_PREFIX}-minion-[0-9]+" \
+                --regexp "${NODE_INSTANCE_PREFIX}-.+" \
```

Comment:
Also pre-existing, but this assumes `${NODE_INSTANCE_PREFIX}` can be used as a regexp and yet doesn't escape the characters in it, so if it had any magic characters such as `.` or `*` or `[`, etc. this might potentially present invalid matches. But this is already in use elsewhere and it's not really easy to fix in shell...

I think my problem here is the amount of shell we let into Kubernetes but right now there's not much we can do in individual PRs to really address that... Sigh.